code/modules/mining/abandoned_crates.dm

```diff
@@ -7,14 +7,14 @@
 	icon_state = "securecrate"
 	var/code = null
 	var/lastattempt = null
+	var/lastattempt2 = null
```

Comment:
Because the game "remembers" which last attempt was input. So when you try your next one, the old one will be there. It would be weird if the player input a 0000 and his next attempt would be zzzz